### PR TITLE
Board render

### PR DESCRIPTION
**Pull request by chloell5**
Scalable functionality for board now mostly in place, merging into a temp branch to protect the consistency of main

### CODE REVIEW

**Comment by chloell5** [comment]
Hopefully this works out okay. If not, I only ruined a temporary branch

File: lib/board.rb [hunk 1]
```diff
@@ -4,22 +4,31 @@ class Board
...
+    num.map do |num|
+      ltr.map do |ltr|
+        coord = ltr + num.to_s
+        cell_grid[coord] = Cell.new(coord)
+      end
+    end
+    cell_grid
+  end
+
+  def cells
```
Comment (hschmid516) [inline lib/board.rb]: Can remove this method now that `@cells` is an instance variable

File: spec/board_spec.rb [hunk 2]
```diff
@@ -84,4 +85,23 @@
       expect(board.valid_placement?(submarine, ["A1", "B1"])).to be false
     end
   end
+
+  context 'Rendering the board' do
```
Comment (hschmid516) [inline spec/board_spec.rb:89]: Printing returns nil so this test wont pass. We could `expect(board.render).to be nil` maybe? Or something else if you can think of something better 😅

**Review by hschmid516 — CHANGES_REQUESTED** [review]
Just two minor changes. Excellent work on the board rendering!!

File: spec/board_spec.rb (hunk 2 above)
Comment (chloell5) [reply spec/board_spec.rb:89]: These tests are disabled for now. I'd ideally like the render method to return the board but I haven't quite figured that out yet

Comment (hschmid516) [reply spec/board_spec.rb:89]: I think to do that you would have to copy the whole method and put it at the end of the method without the `print`

**Pull request merged**